outlook/indico_outlook/calendar.py

```diff
@@ -96,7 +97,14 @@ def _get_reminder(user, event, settings):
     return reminder, reminder_minutes
 
 
-def _update_calendar_entry(entry, settings):
+def _make_calendar_id(event, user, settings):
+    if settings['event_id_cutoff'] != -1 and event.id > settings['event_id_cutoff']:
+        return event.ical_uid
+    else:
+        return f'{settings["id_prefix"]}{user.id}_{event.id}'
```

Comment:
Maybe we can enable [RET505](https://docs.astral.sh/ruff/rules/superfluous-else-return/#superfluous-else-return-ret505)?

Reply:
I tend to prefer having both returns on the same level if they are "equally good".

If one branch errors out / aborts early I'd also return early and then outdent the rest, but for cases like this one I don't think there's a real benefit in outdenting the second return